pkg/router/auth_test.go

```diff
@@ -66,7 +67,9 @@ func TestRouterAuth(t *testing.T) {
 	logger := loggerfactory.GetLogger()
 	testmux := GetRouterWithAuth()
 
-	go httpserver.StartServer(ctx, logger, "test", "8990", testmux)
+	mgr := manager.New()
+
+	go httpserver.StartServer(ctx, logger, mgr, "test", "8990", testmux)
```

Comment:
If we are passing mgr. Do we need to call `go ` here ?

Also we should ask mgr to Wait via defer

Reply:
done

Reply:
done